autogen.sh

```diff
@@ -48,4 +48,11 @@ gtkdocize || exit $?
 autoreconf -fi || exit $?
 
 cd "$olddir"
-test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
+
+# If no arguments passed, will use default arguments for eos-metrics.
+if [ $# -eq 0 ]
+    then
+        test -n "$NOCONFIGURE" || "$srcdir/configure" "--enable-strict-flags" "--enable-gtk-doc" "--enable-gir-doc"
```

Comment:
This doesn't handle the case where someone runs configure directly. Recommend instead removing the flags entirely and modify configure.ac to always configure the way that it currently does when passed these flags. @ptomato is there any reason we would ever want to compile without strict flags, GTK, or GIR enabled? If so, we could change the flags to be "--disable-*" and flip the default behavior in configure.ac.